plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/LongWriteFunction.java

```diff
@@ -27,4 +27,39 @@ default Class<?> getJavaType()
 
     void set(PreparedStatement statement, int index, long value)
             throws SQLException;
+
+    static LongWriteFunction of(int jdbcType, LongWriteFunctionImplementation implementation)
+    {
+        return of(jdbcType, "?", implementation);
+    }
+
+    static LongWriteFunction of(int jdbcType, String bindExpression, LongWriteFunctionImplementation implementation)
```

Comment:
Inline.

Reply:
What do you mean by inline?

Reply:
This is used in 3 places. In those three places just implement `LongWriteFunction` explicitly.
(this happens to be as it was before the change, btw)

Reply:
Done.